### DISCUSSION TITLE
ol 10.1.0 : ERROR] TS2694: Namespace 'RBush' has no exported member 'default'.

### DISCUSSION THREAD

**Discussion by vlo-rte** [openlayers/openlayers#16121] [Q&A]
Do you have an idea of where the problem comes from please ?

**Comment by ahocevar** [comment]
This is  a temporary issue with `rbush`, which should be fixed when `@types/rbush@4` is published. See https://github.com/DefinitelyTyped/DefinitelyTyped/pull/70340

**Reply by vlo-rte** [reply]
OK thanks

**Comment by HaraldKi** [comment]
Pity, but with rbush@4.0.1 it is still a problem, but the PR looks like being worked on. :+1:
